actors/evm/src/lib.rs

```diff
@@ -67,6 +68,7 @@ impl EvmContractActor {
         RT: Runtime,
         RT::Blockstore: Clone,
     {
+        trace!("Entering::constructor=>");
```

Comment:
Would consider using [`#[instrument]`](https://docs.rs/tracing-attributes/latest/tracing_attributes/attr.instrument.html). This would generate a span, not a log, but probably good enough for observability? Would use `cfg_attr` to only enable when a tracing feature is enabled.